pkgs/steam-jupiter/fhsenv.nix

```diff
@@ -67,21 +37,20 @@ let
     mkdir -p ~/.local/state
     >~/.local/state/steamos-session-select echo "$session"
 
-    if [[ -n "$gamescope_pid" ]]; then
-      kill "$gamescope_pid"
-    else
-      >&2 echo "!! Don't know how to kill gamescope"
-      exit 1
-    fi
+    systemctl stop --user gamescope-session
   '';
 
   wrappedSteam = steam-fhsenv.override (extraArgs // {
     extraPkgs = pkgs: (if args ? extraPkgs then args.extraPkgs pkgs else []) ++ [
-      nullOsUpdater nullBiosUpdater
+      dmidecode
+      jovian-stubs
       sessionSwitcher
+
+      # FIXME: figure out how to fix pkexec and readd this
+      # steamos-polkit-helpers
```

Comment:
Can you describe the issue either in this FIXME, or preferably cross-reference a new issue in here?

Reply:
It's just the usual fhsenv SUID nonsense.

Reply:
Yeah, it's more about whoever stumbles on that in a few months, to have a bread crumb to follow.

Reply:
Added a link to the old nixpkgs issue about it.